### DISCUSSION TITLE
Error running pod install (v5.0.0-beta.2)

### DISCUSSION THREAD

**Discussion by jamescrosswell** [llfbandit/record#202] [Q&A]
For some reason in an app I'm building that uses record, I'm getting the following error when trying to debug the app. I think this happened when I upgraded from version 4.4.4 to 5.0.0-beta.2. 

```
[!] CocoaPods could not find compatible versions for pod "record_darwin":
  In Podfile:
    record_darwin (from `Flutter/ephemeral/.symlinks/plugins/record_darwin/macos`)
Specs satisfying the `record_darwin (from `Flutter/ephemeral/.symlinks/plugins/record_darwin/macos`)` dependency were found, but they required a higher minimum deployment target.
```

I'm not too familiar with CocoaPods... anyone else bumped into this or know what the problem might be?

**Comment by jamescrosswell** [comment]
Googling I found [this issue](https://ionic.zendesk.com/hc/en-us/articles/8244535289623-CocoaPods-could-not-find-compatible-versions-for-pod-ReactNativePortals-), which looked quite similar... so I checked `/macos/Podfile` and bumped `platform :osx, '10.14'` to `platform :osx, '10.15'`. I also had to set `MACOSX_DEPLOYMENT_TARGET = 10.15;` in `/macos/Runner.xcodeproj/project.pbxproj`. 

Having done that, I can now build the app again and run it on Mac OS. 

I can't record anything, but that's probably another issue! 😉

**Comment by llfbandit** [comment]
Have a look at the readme to know about macOS setup. Maybe you forgot to enable audio input capability.
